packages/forms-web-app/src/views/final-comment/final-comment.njk

```diff
@@ -0,0 +1,72 @@
+{# todo: change layout based on appeal type when component reused #}
+{% extends "layouts/full-appeal-banner/main.njk" %}
+
+{% from "govuk/components/error-summary/macro.njk" import govukErrorSummary %}
+{% from "govuk/components/textarea/macro.njk" import govukTextarea %}
+{% from "govuk/components/checkboxes/macro.njk" import govukCheckboxes %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+
+{% set title = "Your final comments - Submit final comments - Appeal a planning decision - GOV.UK" %}
+{% block pageTitle %}{{ "Error: " + title if errors else title }}{% endblock %}
+
+{% block content %}
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      {% if errorSummary %}
+        {{ govukErrorSummary({
+          titleText: "There is a problem",
+          errorList: errorSummary,
+          attributes: {"data-cy": "error-wrapper"}
+        }) }}
+      {% endif %}
+      <form action="" method="post" novalidate>
+				<h1 class="govuk-heading-l">
+					What is your final comment?
+				</h1>
+        <p>Your final comment should only respond to the statements from the ((local planning authority / appellant)) or interested parties, and must not include any new evidence.</p>
```

Comment:
confirm with design what is required here?

Reply:
I've confirmed with design, the final comments page should use the "local planning authority" text. The "appellant" text is an alternative version intended for future use on the "LPA" journey. I've updated the template accordingly.